test/langtools/tools/javac/annotations/ModuleImportProcessingTest.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Looks like this test would work better as a "native" annotation processing test.

I recommend look in the `test/langtools/tools/javac/processing` directory hierarchy, specifically for test classes that subclass `JavacTestingAbstractProcessor`. HTH

Reply:
Thanks for the hint. I refactored the test, It looks much better now